src/Clide/Solution/ProjectNode.cs

```diff
@@ -145,11 +145,10 @@ public bool Supports(string capabilities)
         {
             if (!string.IsNullOrEmpty(capabilities))
             {
-                string projectCapabilities = HierarchyNode
-                    .GetProperty<string>((int)__VSHPROPID5.VSHPROPID_ProjectCapabilities);
+                Hierarchy.GetProperty((uint)VSConstants.VSITEMID.Root, (int)__VSHPROPID5.VSHPROPID_ProjectCapabilities, out object projectCapabilities);
 
-                if (!string.IsNullOrEmpty(projectCapabilities))
-                    return ExpressionEvaluator.GetValue().EvaluateExpression(capabilities, projectCapabilities);
+                if (!string.IsNullOrEmpty(projectCapabilities.ToString()))
```

Comment:
var projectCapabilities = projectCapabilitiesAsObject as string; // avoids the duplicate ToString and the potential NullReferenceException